cache/gcs/gcs.go

```
@@ -47,8 +47,16 @@ func New(config dict.Dicter) (cache.Interface, error) {
 	}
 
 	gcsCache.Basepath, err = config.String(ConfigKeyBasepath, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
 	defaultMaxZoom := uint(tegola.MaxZ)
+
 	gcsCache.MaxZoom, err = config.Uint(ConfigKeyMaxZoom, &defaultMaxZoom)
+	if err != nil {
+		log.Fatal(err)
+	}
```

please return the error rather than log fatal.